runbotics/runbotics-orchestrator-ui/src/src-app/layouts/MainLayout/NavBar/usePublicSections.ts

```diff
@@ -21,7 +21,7 @@ export const usePublicSections = (): Section[] => {
                 {
                     title: translate('Nav.Items.Processes'),
                     icon: CallSplitIcon,
-                    href: '/app/processes',
+                    href: '/app/processes/collections',
```

Comment:
I assume that we want to have process collections view as a default, so what do you think about switching also tabs order in general Process view?